### PR TITLE
Visit people, me and id pages

### PR DESCRIPTION
**Pull request by shiroyasha**

### CODE REVIEW

File: src/people_page.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+echo "---------------------------------"
+echo "People Page"
+echo "---------------------------------"
+
+page_url="https://${ORGANIZATION_USERNAME}.semaphoreci.com/projects/${SEMAPHORE_PROJECT_ID}/people"
```
Comment (radwo) [inline src/people_page.sh:7]: Do we have SEMAPHORE_PROJECT_NAME ?

Comment (shiroyasha) [reply src/people_page.sh:7]: no, but that can be hardcoded if necessaery, we know that this project is called cont-monitoring

Comment (radwo) [reply src/people_page.sh:7]: @shiroyasha `project_id` is more like our url for easy access projects by id, and `project_name` is more clients url ;)

So I think that we should check clients url.

Comment (shiroyasha) [reply src/people_page.sh:7]: I agree. While writting the impl yesterday, that url was not yet accessible (or maybe the people link was still containing the project id).

**Pull request merged**